src/Aspire.Hosting/Health/ResourceNotificationHealthCheckPublisher.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Aspire.Hosting.Health;
+
+internal class ResourceNotificationHealthCheckPublisher(DistributedApplicationModel model, ResourceNotificationService resourceNotificationService) : IHealthCheckPublisher
+{
+    public async Task PublishAsync(HealthReport report, CancellationToken cancellationToken)
+    {
+        _ = model;
+        _ = resourceNotificationService;
```

Comment:
delete
```suggestion
```

Reply:
I didn't want these two variables getting too big for their boots so I decided to slap them down up front in this class .... fixed.